src/System.Composition.Hosting/src/ILLinkTrim.xml

```diff
@@ -0,0 +1,22 @@
+<linker>
+  <assembly fullname="System.Composition.Hosting">
+    <type fullname="System.Composition.Hosting.Providers.ExportFactory.ExportFactoryExportDescriptorProvider">
```

Comment:
Folks this isn't ready for review. I'm just submitting for test coverage at the moment.